public.env.js

```diff
@@ -1,14 +1,15 @@
 const info = `
-SERVER_PORT=7000
+SERVER_PORT=8000
 DATABASE_PORT=3306
-DATABASE_NAME=database
+DATABASE_NAME=blogger
 DATABASE_HOST=localhost
 DATABASE_USER=root
 DATABASE_PASSWORD=
-DATABASE_TABLE_NAME=table
-DB_URI=mysql://root@localhost:3306/database
-JWT_SECRET_TOKEN=abcdefghij
-JWT_SECRET_TOKEN_EXPIRE_TIME=20min
+DATABASE_TABLE_NAME=users
+DB_URI=mysql://root@localhost:3306/blogger
+JWT_SECRET_TOKEN=khondokhondocetonagulochetonahinhoyeroysarthohasilermohautsobesottorakorevoy
+JWT_SECRET_TOKEN_EXPIRE_TIME=86400
 JWT_SECRET_TOKEN_ALGORITHM=HS256
```

Comment:
Is this visible intentionally?

Reply:
Yes, this is intentionally visible. My actual .env file is not visible.